model_bakery/generators.py

```diff
@@ -132,6 +153,18 @@ def gen_integer():
     default_mapping[PositiveBigIntegerField] = _make_integer_gen_by_range(
         PositiveBigIntegerField
     )
+if DecimalRangeField:
+    default_mapping[DecimalRangeField] = random_gen.gen_pg_numbers_range(Decimal)
+if IntegerRangeField:
+    default_mapping[IntegerRangeField] = random_gen.gen_pg_numbers_range(int)
+if BigIntegerRangeField:
+    default_mapping[BigIntegerRangeField] = random_gen.gen_pg_numbers_range(int)
```

Comment:
Python 3's `int` type is clever than Py2 because it's the previous `long` type, but with the same `int` name. The [PEP-237](https://www.python.org/dev/peps/pep-0237/) has the implementation details on that. 

So, we don't need to worry and we can rely on the `int` type.